regex-assembly/include/932130.ra

```diff
@@ -1,13 +1,13 @@
 ##! Please refer to the documentation at
 ##! https://coreruleset.org/docs/development/regex_assembly/.
 
-\$\(.*\)
-\$\{.*\}
-<\(.*\)
->\(.*\)
-\$\(\(.*\)\)
+\$\(.+?\)
```

Comment:
This is a semantic change from "0 or more" to "1 or more". Is that intended?

Reply:
No. Changing that back.